### DISCUSSION TITLE
.wgt file to install.

### DISCUSSION THREAD

**Discussion by Nexow91** [jhassan8/crunchyroll-tizen#2] [General]
Hello thank you first of all for the effort and work to be able to install crunchyroll on tizen I wanted to ask is the project now complete ? If I remember correctly, you needed a ".wgt" file to install the app through the Tizen Studio. 

Thank you

**Comment by jhassan8** [comment]
hi the main flow is complete, some features are missing, such as the next episode at the end, forward and rewind, quote list and the search option... I'll see if I can add them quickly.

You can generate the ".wgt" with Tizen Studio, but now I'll upload the file.
Let me know if you have any errors when using it on TV, there are many incompatibilities in tizen versions... regards
